test/external/druidExternal.mocha.js

```diff
@@ -1697,6 +1714,51 @@ describe("DruidExternal", () => {
       });
     });
 
+    // todo: this doesn't work.
+    it.skip("works with custom transform split", () => {
+      var ex = $('wiki')
+        .split($('time').customTransform('makeFrenchCanadian').cast('STRING').concat($('time').customTransform('makeExcited').cast('STRING')), 'FancyTime');
+
```

Comment:
you can not combine (concat) two transforms into one in Druid. make this into two tests one with `$('time').customTransform('makeFrenchCanadian')` and another with `$('time').customTransform('makeExcited').cast('STRING')`